src/python-common/ceph/tests/test_drive_group.py

```diff
@@ -20,25 +20,13 @@
             }
         ]
     ),
-    (
-        {  # old style json
-            'testing_drivegroup':
```

Comment:
e.g. the docs still make use of this format.

Reply:
I'll update the docs either in this or in a separate PR